packages/components/src/styles/global.css

```diff
@@ -12,4 +12,20 @@
   --tw-ring-shadow: 0 0 #0000;
 }
 
+/**
+ * This injects Tailwind's base styles and any base styles registered by
+ * plugins.
+ */
 @tailwind base;
+
+/**
+ * This injects Tailwind's component classes and any component classes
+ * registered by plugins.
+ */
+@tailwind components;
```

Comment:
My read of https://tailwindcss.com/docs/extracting-components#extracting-component-classes-with-apply is that this is where the `@apply`-ed stuff goes.

But I thought we were already getting those in the compiled css? 🤔 

Is it possible we're now duplicating anything from `@apply`?

Reply:
I think Jin noted that the `components.css` class contains really few things: https://unpkg.com/tailwindcss@2.2.17/dist/components.css. My read of that section was that you can use `@layer` to control specificity, but it doesn't overlap with the content of `@tailwind components`

Reply:
And if the component classes are not being used, they should be purged out

Reply:
Yeah you're right. Looks like `@apply` stuff gets inlined right into the .module.css file